sovrin_node/test/upgrade/test_migration_tool.py

```diff
@@ -45,3 +45,12 @@ def testMigrateTimesOut(monkeypatch):
 
     with pytest.raises(TimeoutError):
         migration_tool.migrate(TEST_VERSION, TEST_NEW_VERSION, TEST_TIMEOUT)
+
+def testGetMigrationScripts():
+    try:
+        res = migration_tool._get_migration_scripts(migration_tool._get_current_platform())
+    except Exception as e:
+        pytest.fail("Unexpected error: {}".format(e))
+    else:
+        assert type(res) is list
```

Comment:
Is there a guarantee that `res` is always a `list` and not a subclass of `list`? If not then replace by `assert isinstance(res, list)`

Reply:
yes, currently list contructor is used directly https://github.com/andkononykhin/indy-node/blob/5fa2f762d479e68a8e02e2b59b94b4c2829284d8/sovrin_node/utils/migration_tool.py#L76
I think it could be adjusted in future if something changed and test fails.